### PR TITLE
Update InSpec project members

### PR DESCRIPTION
**Pull request by james-stocks**
Signed-off-by: James Stocks <jstocks@chef.io>

### CODE REVIEW

File: projects/inspec.md [hunk 1]
```diff
@@ -10,16 +10,20 @@ The InSpec project includes InSpec as well as several resource packs and plugins
...
 #### Project Owners
 
-Clinton Wolfe
-  - Github: [clintoncwolfe](https://github.com/clintoncwolfe)
-  - Slack: @cwolfe
+James Stocks
+  - Github: [james-stocks](https://github.com/james-stocks)
+  - Slack: @Stocksy
 
-Ryan Davis
```
Comment (tas50) [inline projects/inspec.md:17]: Ryan needs to move to the Alums section

Comment (james-stocks) [reply projects/inspec.md:17]: Good call, thanks!

File: projects/inspec.md [hunk 2]
```diff
@@ -10,22 +10,32 @@ The InSpec project includes InSpec as well as several resource packs and plugins
...
   - Slack: @rdavis
 
-#### Project Approvers
-
-#### Project Reviewers
-
 ### Project Repositories
 
 - [inspec](https://github.com/inspec/inspec)
-- [inspec-gcp](https://github.com/inspec/inspec-gcp)
```
Comment (tas50) [inline projects/inspec.md:28]: If we're not maintaining this anymore we should note that on the repo as well

Comment (james-stocks) [reply projects/inspec.md:28]: The repo is listed as maintained by https://github.com/chef/chef-oss-practices/blob/master/projects/inspec-cloud.md 

I don't see the repo referencing this team/project

Comment (tas50) [reply projects/inspec.md:28]: k

File: projects/inspec.md [hunk 3]
```diff
@@ -10,22 +10,32 @@ The InSpec project includes InSpec as well as several resource packs and plugins
...
+- [train](https://github.com/inspec/train)
```
Comment (tas50) [inline projects/inspec.md:38]: Train is a foundation project

Comment (james-stocks) [reply projects/inspec.md:38]: Oops - removed it from the list. Thanks for steering me right!

**Pull request merged**